torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1426,16 +1439,14 @@ bool trackSingleGradSumToSizeToOutputs(
 }
 
 void FuseGraph(std::shared_ptr<Graph>& graph) {
-  if (canFuseOnCPU() || canFuseOnGPU()) {
```

Comment:
It's still a good idea to keep this `if`, because if you don't have type information you will always assume the fusion is possible, while in reality none of the devices we support can work with fused kernels. This might also cause some regressions if we have tests that disable fusions.